contracts/Homework.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.3;
+
+import "./IterableMap.sol";
+
+contract Homework {
+    IterableMap map = new IterableMap();
```

Comment:
This line deploys a new `IterableMap` contract and stores its address in `map` field. This happens during deployment of `Homework` contract. 

You can [see it in action on Etherscan](https://ropsten.etherscan.io/tx/0x8ee9fd42645a2f010c27c1fe81c5ba83d00e1ebe51f61997d29e23f813193f20#internal). In the same transaction that deployed your `0x7930...` `Homework` contract, an instance of `IterableMap` was deployed at `0x7DB3...`. 

This is probably not what you intended to happen. We usually **inherit** from the other contract to get it's functionality.